eftest/src/eftest/runner.clj

```diff
@@ -98,7 +118,8 @@
                        (defaults to eftest.report.progress/report)
     :capture-output? - if true, catch test output and print it only if
                        the test fails (defaults to true)
-    :fail-fast?      - stop after first failure or error"
+    :fail-fast?      - stop after first failure or error
+    :test-warn-length - print warning for any test that exceeds this value in milliseconds Measured duration includes each-fixtures"
```

Comment:
Line width should be max 80 characters, and docstring should have correct punctuation.

Reply:
Fixed